httpbp/middlewares.go

```diff
@@ -510,3 +520,31 @@ func (rr *responseRecorder) WriteHeader(code int) {
 	rr.ResponseWriter.WriteHeader(code)
 	rr.responseCode = code
 }
+
+const (
+	RedditDebug        = "X-Reddit-Debug"
+	RedditDebugEnabled = "enabled"
+	RedditK8sMeta      = "X-Reddit-K8S-Meta"
+	redditK8sMetaFmt   = "%s-%s-%s"
```

Comment:
Don't move format strings to constants.  This makes it much harder to understand at the use-site.